dist/style/header.css

```diff
@@ -63,7 +63,7 @@ input[type="checkbox"][id^="cb"] {
     background-color: rgba(0,0,0,0.5);
 }
 
-@media screen and (max-width: 1000px) {
```

Comment:
Just so that I'm fully aware of what's going on, what prompted this change? ^^

Reply:
because now more items are in the header it needs to turn into a toggable sidebar sooner otherwise some items get pushed off-screen